src/Navigation/Navigation.tsx

```diff
@@ -1,16 +1,19 @@
 import React from 'react'
 import './Navigation.scss'
+import {Link} from 'react-router-dom'
 
 const Navigation = () => {
   return (
     <nav className="navigation-bar">
       <h1 className="logo">
-        <div>
-          Art From The 
-        </div>
-        <div className="logo-accent">
-          Heart
-        </div>
+        <Link className='title-link' to='/' >
+          <div className='art-from-the'>
+            Art From The 
+          </div>
+          <div className="logo-accent">
+            Heart
+          </div>
+        </Link>
```

Comment:
Thank you for making this a link back to the home page! Excellent navigation for our users!